README.md

```diff
@@ -1 +1,52 @@
 # products_aggregator
+
+## Table of contents
+
+
+* [Getting Started](#Getting-Started)
+  * [Required packages](#Required-packages)
+  * [Install project](#Install-project)
+* [Documentation](#Documentation)
+  * [Run server](#Run-server)
+  * [Run unit tests](#Run-unit-tests)
+
+### Getting Started
+
+#### Required packages
+
+**Products_aggregator** use MariaDB. Please, install next packages:
+`python-pip python-dev virtualenv mariadb-server libmariadbclient-dev 
+libssl-dev`
+
+Also we recommend to use python 3.6.
+
+#### Install project
+
+1. Complete security script by running:
+`sudo mysql_secure_installation`
+2. Create database, user and grant privileges
+for database to user. You can use this [manual](https://www.digitalocean.com/community/tutorials/how-to-use-mysql-or-mariadb-with-your-django-application-on-ubuntu-14-04)
```

Comment:
What privileges should be granted to the user?

Reply:
> What privileges should be granted to the user?

I fix it. Thank you for identifying this issue.